### PR TITLE
FAQ page

### PR DESCRIPTION
**Pull request by Louismaestre**
- The FAQ page is composed of questions and answers to make it easier for users to use the application.

- Fixed object creation problems in the container-profile section by adding the “Create object” button.

- Fixed a problem with the name of a server function that displays the list of objects

Coverage:

Capture d’écran 2024-09-15 à 14 37 53

FAQ Page:

Capture d’écran 2024-09-12 à 18 26 25

Capture d’écran 2024-09-12 à 18 26 31

Bouton création d'objet:

Capture d’écran 2024-09-15 à 15 38 48

### CODE REVIEW

File: Configurateur/front/lib/screens/company-profil/container-profil.dart [hunk 1]
```diff
@@ -599,189 +596,203 @@ class _ContainerProfilPageState extends State<ContainerProfilPage> {
...
+                    ],
+                  ),
+                ),
               ),
-            ),
-          ),
-        ],
-      ),
-    );
+            ]));
```
Comment (invpuppet) [inline Configurateur/front/lib/screens/company-profil/container-profil.dart:796]: Why remove this

**Review by invpuppet — CHANGES_REQUESTED** [review]

File: Configurateur/front/lib/components/container.dart [hunk 2]
```diff
@@ -161,9 +161,16 @@ class ContainerCards extends StatelessWidget {
...
+                    icon: const Icon(Icons.chevron_right),
+                    onPressed: () {
+                      if (container.id != null) {
+                        storageService.writeStorage(
+                          'containerId',
+                          container.id.toString(),
+                        );
+                      }
+                      context.go(page);
+                    }),
```
Comment (H-chauvet) [inline Configurateur/front/lib/components/container.dart:173]: here again, missing comma (,)

File: Configurateur/front/lib/screens/faq/faq.dart [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+                          answer: faq["answer"]!,
+                          linkText: faq["linkText"]!,
+                          page: faq["page"]!,
+                          isOpen: openedQuestionIndex == index,
+                          questionNumber: index,
+                          onToggle: () {
+                            setState(() {
+                              openedQuestionIndex =
+                                  openedQuestionIndex == index ? null : index;
+                            });
```
Comment (H-chauvet) [inline Configurateur/front/lib/screens/faq/faq.dart:160]: Same again missing comma (,)

File: Configurateur/front/lib/screens/faq/faq.dart [hunk 4]
```diff
@@ -0,0 +1,287 @@
...
+                          },
+                        );
+                      }).toList(),
```
Comment (H-chauvet) [inline Configurateur/front/lib/screens/faq/faq.dart:163]: Same again missing comma (,)

File: Configurateur/front/lib/screens/faq/faq.dart [hunk 5]
```diff
@@ -0,0 +1,287 @@
...
+                    ],
+                  ),
+                ),
+              ),
+            ),
+          ]),
```
Comment (H-chauvet) [inline Configurateur/front/lib/screens/faq/faq.dart:169]: Same again

**Review by H-chauvet — CHANGES_REQUESTED** [review]

**Review by invpuppet — APPROVED** [review]

**Review by H-chauvet — APPROVED** [review]

**Pull request merged**